docs/compare.md

```diff
@@ -21,6 +21,7 @@ Here is the review of generic features, such as supported MongoDB versions, open
 | Kubernetes conformance | Various versions are tested | No guarantee   | No guarantee       | No guarantee                       | No guarantee                 |
 | Cluster-wide mode | Yes                          | Not an operator    | Enterprise only    | Yes                                | Yes                          |
 | Network exposure  | Yes                          | Yes                | No, only through manual config | No                     | Yes                          |
+| Web-based GUI     | [Percona Everest](https://docs.percona.com/everest/index.html) | :no_entry_sign: | kubedb-ui | :no_entry_sign:| [MongoDB Atlas](https://www.mongodb.com/docs/atlas/getting-started/)|
```

Comment:
1. is web-based gui is the best we can come up with?
2. `kubedb-ui` - do you have any links for that?
3. MongoDB also has OpsManager. And I would mention it vs Atlas as it is the tool to run DBs on k8s, not atlas.